src/templates/image-text/image-text.js

```diff
@@ -83,10 +102,14 @@ function ImageText({ slide, content, run, slideDone }) {
     rootClasses = rootClasses.concat(' animated-header');
   }
 
+  if (shadow) {
+    rootClasses = rootClasses.concat(' shadow');
```

Comment:
All these string concats with spaces are a bit confusing. Maybe use arrays of classes and join them instead.
https://www.w3schools.com/jsref/jsref_join.asp